### ISSUE TITLE
Make 'Open in new window' answer match guidance 

### ISSUE THREAD

**Issue by jbuller** [ministryofjustice/record-a-goose-sighting#27]
Updated Design System guidance is to only use text not an icon to designate a link that opens in a new window. https://design-system.service.gov.uk/styles/typography/#links
This conflicts with the text of the answer to this exercise.

**Comment by BNewing** [comment]
Oh great! I've got a suggestion for some new text in a PR for this. Let me know what you think :) I might add in something about that you should only do opening pages in new tabs sparingly too.

**Comment by jbuller** [comment]
Yes, looks good. I agree that like the guidance you can be stronger on saying new windows should be avoided in general.
